com.abagames.noiz2sa.yaml

```diff
@@ -0,0 +1,90 @@
+app-id: com.abagames.noiz2sa
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: run-noiz2sa.sh
+rename-desktop-file: noiz2sa.desktop
+rename-icon: noiz2sa
+rename-appdata-file: noiz2sa.appdata.xml
+finish-args:
+- "--device=dri"
+- "--share=ipc"
+- "--socket=pulseaudio"
+- "--socket=x11"
+modules:
+- shared-modules/glu/glu-9.json
+- shared-modules/SDL/SDL-1.2.15.json
+- shared-modules/SDL/SDL_mixer-1.2.12.json
+- name: bulletml
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: http://http.debian.net/debian/pool/main/b/bulletml/bulletml_0.0.6.orig.tar.gz
```

Comment:
`ldconfig: /app/lib/libbulletml.so.0d2 is not a symbolic link` while running.

Looks like this symlink is not created: `libbulletml.so.0d2 -> libbulletml.so.0d2.0`

The patches looks from debian which has the symlink. Can you please check if the source or any of them needs updating to fix this? Thanks.

Reply:
Ah found the issue, you're doing plain `cp` in the build commands which doesn't preserve the link, change it to `cp -a` `cp -a src/lib*.so* /app/lib/`